features/README.md

```diff
@@ -15,7 +15,9 @@ for even older versions.
 
 ## Install
 
+```shell
```

Comment:
This produces excessive indentation
![A06D6585-9C7B-41DC-9FEB-B2EE7D7650CE](https://github.com/rspec/rspec-rails/assets/6916/1306e1cd-b8a9-435e-9625-bd5ee3d6fc9c)

I understand and am very much appreciated how you keep the content intact and only fix the layout.
If you want to keep blame etc clean of the formatting changes you may add those commits to [ignore revs](https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/).

Reply:
Before the change it seems it was stripping the whitespace from the left, but now it doesn’t
![5CE411DD-9F19-4879-900C-E23321F5DA54](https://github.com/rspec/rspec-rails/assets/6916/a76b3958-a7d0-4768-b323-e1c140041f61)

Reply:
Ok, I can fix that. I just need to outdent the codeblocks as well.

Reply:
In the updated version I just pushed, all code blocks that were extra-indented as a result of my updates now have no extra spaces at the front of them.

I will go back and check the other repos I pushed as well. rspec-core had minimal changes, so I didn't introduce any extra indentation, but looking at it there are places that exists (as well as places that use a '$' as indicator of a console command, although the syntax highlighting specifies Ruby. I have not touched these).